src/pybind/mgr/telemetry/module.py

```diff
@@ -939,6 +1036,117 @@ class EndPoint(enum.Enum):
         ceph = 'ceph'
         device = 'device'
 
+    def collection_delta(self) -> List[Collection]:
+        '''
+        Find collections that are available in the module, but are not in the db
+        '''
+        new_collection : List[Collection] = []
+
+        for c in MODULE_COLLECTION:
+            for k, v in c.items():
```

Comment:
since c is a dict, it seems like a loop is unnecessary here and in a few other places - you could look up c['name'] instead